<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostEdit</class>
 <widget class="QWidget" name="PostEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border-radius: 10px;
border: 1px solid #555;
background-color: rgb(14, 15, 18);
}
#frame_09, #frame_99, #frame_88{
border: none;
}

#title, #description {
	border-radius: 5px;
	color:rgba(255, 255, 255, 150);
	padding-bottom:4px;
	background-color: #2C2828;
}

#horizontalFrame{
background-color: none;
border : none;
}

#backBtn, #confirmBtn{	
border: 1px solid #555;
font-weight: bold;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 44, 85, 100), stop:1 rgba(0, 0, 0, 255));
border-radius: 5px;
color: #fff;
}

#deletePostBtn{
border: 1px solid #555;
font-weight: bold;
background-color: rgba(254, 44, 85, 100);
border-radius: 10px;
color: #fff;

}
#backBtn::hover, #confirmBtn::hover{	
font-weight: bold;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 44, 85, 150), stop:1 rgba(0, 0, 0, 255));
}

#backBtn::pressed, #confirmBtn::pressed{	
font-weight: bold;
padding-left:5px;
padding-top:2px;
background-color: #2C2828;
}


#deletePostBtn:hover{
font-weight: bold;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(254, 44, 85, 150), stop:1 rgba(0, 0, 0, 255));

}
#deletePostBtn:pressed{
border: 1px solid #555;
padding-left:6px;
padding-top:4px;
		border-radius: 8px;

	background-color: #2C2828;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_09" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>525</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_88">
        <property name="minimumSize">
         <size>
          <width>226</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="thumbnailLabel">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>375</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>375</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources.qrc">:/images/images/thumbnail/DougSinglemann--CrowRiver2016.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePostBtn">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>  D E L E T E   P O S T</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/icons/feather/x-octagon.svg</normaloff>:/icons/icons/feather/x-octagon.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_99">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="title">
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>265</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="description">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>265</width>
             <height>121</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>265</width>
             <height>121</height>
            </size>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Description ...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="horizontalFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>271</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>271</width>
             <height>51</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QPushButton" name="backBtn">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>B A C K</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/icons/feather/arrow-left-circle.svg</normaloff>:/icons/icons/feather/arrow-left-circle.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="confirmBtn">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>S A V E</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/icons/icons/feather/save.svg</normaloff>:/icons/icons/feather/save.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
